addons/website_sale/views/snippets.xml

```diff
@@ -61,16 +61,22 @@
     </xpath>
 </template>
 
+<template id="website_sale_search_box" inherit_id="website.website_search_box">
```

Comment:
"website_sale_search_box" seems to be a too generic name for the specific stuff it does. "website_sale_products_search_box" maybe ?